integration/integration.go

```diff
@@ -17,27 +17,41 @@ type Integration struct {
 	Name               string    `json:"name"`
 	ProtocolVersion    string    `json:"protocol_version"`
 	IntegrationVersion string    `json:"integration_version"`
-	Data               []*Entity `json:"data"`
+	Entities           []*Entity `json:"data"`
 	prettyOutput       bool
 	writer             io.Writer
 }
 
 // New creates new integration with sane default values.
-func New(name, version string) (*Integration, error) {
-	return NewBuilder(name, version).Build()
+func New(name, version string, args interface{}) (*Integration, error) {
+	return NewBuilder(name, version).ParsedArguments(args).Build()
 }
 
-// NewWithArgs creates new integration with given arguments and sane default values.
-func NewWithArgs(name, version string, args interface{}) (*Integration, error) {
-	return NewBuilder(name, version).ParsedArguments(args).Build()
+// DefaultEntity retrieves default entity to monitorize.
+func (i *Integration) DefaultEntity() *Entity {
```

Comment:
Proposal: maybe name it  `LocalEntity`?

Reply:
We finally decided going for your proposal @mariomac. Are u going to change it here in this PR @varas ?

Reply:
Nope, next one https://github.com/newrelic/infra-integrations-sdk/pull/73